cpu/stm32/periph/gpio_ll_irq.c

```diff
@@ -42,25 +42,19 @@
 
 #if defined(EXTI_SWIER_SWI0) || defined(EXTI_SWIER_SWIER0)
 #  define EXTI_REG_SWIER        (EXTI->SWIER)
-#endif
-
-#if defined(EXTI_SWIER1_SWI0) || defined(EXTI_SWIER1_SWIER0)
+#elif defined(EXTI_SWIER1_SWI0) || defined(EXTI_SWIER1_SWIER0)
```

Comment:
That shouldn't make a difference, as only ever one of the if directives can be true.

In fact, if both would be true the same symbol would be redefined with a different value, which would cause a compilation failure caught by the CI.

Reply:
OK, if that is more readable, please go ahead.